plugins/uefi-dbx/fu-plugin-uefi-dbx.c

```diff
@@ -41,6 +41,37 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 	return TRUE;
 }
 
+static void
+fu_plugin_add_security_attrs_runtime (FuPlugin *plugin, FuUefiDbxFile *dbx, FuSecurityAttrs *attrs)
+{
+	g_autoptr(GError) error_local = NULL;
+	g_autoptr(FwupdSecurityAttr) attr = NULL;
+
+	/* create attr */
+	attr = fwupd_security_attr_new (FWUPD_SECURITY_ATTR_ID_UEFI_DBX_ESP);
+	fwupd_security_attr_set_plugin (attr, fu_plugin_get_name (plugin));
+	fwupd_security_attr_add_flag (attr, FWUPD_SECURITY_ATTR_FLAG_RUNTIME_ISSUE);
+	fu_security_attrs_append (attrs, attr);
+
+	/* no binary blob */
+	if (!fu_plugin_get_enabled (plugin)) {
+		fwupd_security_attr_set_result (attr, FWUPD_SECURITY_ATTR_RESULT_NOT_FOUND);
+		fwupd_security_attr_set_url (attr, "https://github.com/fwupd/fwupd/wiki/Missingdbx");
+		return;
+	}
+
```

Comment:
before even digging into the ESP, since this check has a dependency on `pesign` being present, I think you should have a check right now whether the check can even be run, and set a result accordingly.